angr/analyses/identifier/runner.py

```diff
@@ -75,10 +75,10 @@ def _get_recv_state(self):
             entry_state.unicorn.max_steps = 10000
 
             pg = self.project.factory.simgr(entry_state)
+            stop_addr = self.project._simos.syscall_from_number(2).addr
             num_steps = 0
             while len(pg.active) > 0:
-                syscall = self.project._simos.syscall_table.get_by_addr(pg.one_active.addr)
```

Comment:
I think what I was looking for with this comment is some sort of `self.project.engine_for(some_addr)`, then you could get whatever info from that engine in the end.